### DISCUSSION TITLE
news模块无法更新

### DISCUSSION THREAD

**Discussion by AltiaEvan** [NSRingo/.github#4] [Q&A 问答] [answered]
![image](https://github.com/user-attachments/assets/ee105926-2a9b-4b1e-888a-547c07332a74)

**Comment by VirgilClyne** [comment, marked as answer]
`Quantumult X`所使用的「News」的`snippet`同时包含`分流`和`重写`两种资源
请开启`资源解析器`后，将「News」的`snippet`同时添加至`引用资源 - 分流`和`引用资源 - 重写`
